.gitmodules

```diff
@@ -1,15 +1,15 @@
 [submodule "sonic-swss-common"]
 	path = src/sonic-swss-common
-	url = https://github.com/Azure/sonic-swss-common
+	url = https://github.com/barefootnetworks/sonic-swss-common
```

Comment:
cannot use barefootnetwork repo for sonic-swss-common

Reply:
This is just temporary until https://github.com/Azure/sonic-swss-common/pull/205 is merged. Will revert to azure repos before merging

Reply:
merged